cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -6,131 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-`{odo-title}` is a CLI tool for creating applications on {product-title} and Kubernetes. With `{odo-title}`, you can write, build, and debug applications on a cluster without the need to administer the cluster itself.
-Creating deployment configurations, build configurations, service routes and other {product-title} or Kubernetes elements are all automated by `{odo-title}`.
+Red Hat OpenShift Developer CLI (`odo`) is a tool for creating applications on {product-title} and Kubernetes. With `{odo-title}`, you can develop, test, debug and deploy microservices-based applications on a Kubernetes cluster without having a deep understanding of the platform.
 
-Existing tools such as xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-about-cli_cli-developer-commands[`oc`] are operations-focused and require a deep understanding of Kubernetes and {product-title} concepts. `{odo-title}` abstracts away complex Kubernetes and {product-title} concepts allowing developers to focus on what is most important to them: code.
+`{odo-title}` follows a _create and push_ workflow. As a user, when you _create_, the information (or manifest) is stored in a configuration file. When you _push_, the corresponding resources are created on the Kubernetes cluster. All of this configuration is stored in the Kubernetes API for seamless accessibility and functionality.
 
-[id="odo-key-features"]
-== Key features
+`{odo-title}` uses _deploy and link_ commands to link components and services together. `{odo-title}` achieves this by creating and deploying services based on Kubernetes Operators in the cluster. Services can be created using any of the operators available on the Operator Hub. After linking a service, `odo` injects the service configuration into the service. Your application can then use this configuration to communicate with the Operator-backed service.
```

Comment:
I think it is `uses _service and link_`, as this is the `odo service create` command used to anage the services, not the `odo deploy` one (`odo deploy` is used for the outer loop)

Reply:
After linking a service, `odo` injects the service configuration into the ~~service~~ component (or application?).